search/api.py

```diff
@@ -0,0 +1,261 @@
+"""
+Functions for search
+"""
+from itertools import islice
+import logging
```

Comment:
Is this our convention or a PEP 8 requirement? I'm happy to make the change, just haven't heard of this before